### PR TITLE
Fixed grammar in Glossary. Updated outline

### PR DESCRIPTION
**Pull request by FieldmanJR1**

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -42,47 +42,43 @@ Extrinsic properties can affect the behavior of a function or method, but they a
...
 
 ## Governance
 
-The process of determining what changes to the network are permissible, such as modifications to code or movement of funds.  The governance system in Polkadot is on-chain and revolves around stakeholder voting, i.e. the majority of the stake (DOTs) determines the direction of the network.
+The process of determining what changes to the network are permissible, such as modifications to code or movement of funds. The governance system in Polkadot is on-chain and revolves around stakeholder voting, i.e., the majority of the stake (DOTs) determines the direction of the network.
 
 ## LIBP2P
 
-An open-source library for encrypted peer-to-peer communications and other networking functionality.  More information at: [https://libp2p.io/](https://libp2p.io/)
+An open-source library for encrypted peer-to-peer communications and other networking functionality. More information at [https://libp2p.io/](https://libp2p.io/)
```
Comment (xcthulhu) [inline docs/glossary.md:61]:
```suggestion
An open-source library for encrypted peer-to-peer communications and other networking functionality. More information at <https://libp2p.io/>
```

See https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses

File: docs/glossary.md [hunk 2]
```diff
@@ -42,47 +42,43 @@ Extrinsic properties can affect the behavior of a function or method, but they a
...
+Nodes that select a set of validators. A certain amount of DOTs must be staked in order to do so, which may be lost if the validator behaves badly. This forces nominators to carefully select validators.
 
 ## NTP
 
-Networking protocol designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently it is required that validators' local clocks stay reasonably in sync, so you should be running NTP or a similar service
-
-## On-chain governance
-
-Governance of a blockchain which is controlled by mechanisms controlled by the blockchain.  On-chain governance allows for decisions can be made in a transparent manner.  Note that there are a variety of different algorithms for making these decisions, such as simple majority voting or identity-based quadratic voting.
+Networking protocol is designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently, it is required that validators' local clocks stay reasonably in sync, so you should be running NTP or a similar service.
```
Comment (xcthulhu) [inline docs/glossary.md:81]: Mention that NTP stands for "Network Time Protocol"

File: docs/glossary.md [hunk 3]
```diff
@@ -116,19 +112,19 @@ A session "key" is a BLS key for GRANDPA, a sr25519 key for BABE, and eventually
 
 ## Staking
 
-"Reserving" tokens (for GGX network, GGX tokens) which are put up as "collateral" for a chance to produce a valid block (and thus obtain a block reward).  Validators and nominators (who back validators through NPoS) together stake their GGX tokens in order to add blocks.
+"Reserving" tokens (for GGX network, GGX tokens) that are put up as "collateral" for a chance to produce a valid block (and thus obtain a block reward). Validators and nominators (who back validators through NPoS) together stake their GGX tokens in order to add blocks.
 
 ## Stash account
```
Comment (xcthulhu) [inline docs/glossary.md:117]:
```suggestion
## Stash Account
```

File: docs/glossary.md [hunk 4]
```diff
@@ -116,19 +112,19 @@ A session "key" is a BLS key for GRANDPA, a sr25519 key for BABE, and eventually
...
 
-In Golden Gate, a stash account is the primary account that holds a user's GGX tokens and is used for staking and nominating on the network. When a user wants to participate in staking and nominating activities, they must first transfer their GGX tokens to their stash account.
+In Golden Gate, a stash account is a primary account that holds a user's GGX tokens and is used for staking and nominating on the network. When a user wants to participate in staking and nominating activities, they must first transfer their GGX tokens to their stash account.
 
 ## State transition function
```
Comment (xcthulhu) [inline docs/glossary.md:121]:
```suggestion
## State Transition Function
```

File: docs/glossary.md [hunk 5]
```diff
@@ -42,47 +42,43 @@ Extrinsic properties can affect the behavior of a function or method, but they a
...
 
-An open-source library for encrypted peer-to-peer communications and other networking functionality.  More information at: [https://libp2p.io/](https://libp2p.io/)
+An open-source library for encrypted peer-to-peer communications and other networking functionality. More information at [https://libp2p.io/](https://libp2p.io/)
 
 ## Liveness
 
-The property of a distributed system that it will eventually come to some sort of consensus.  A system stuck in an infinite loop would not be considered live, even if computations are taking place; a system which eventually provides a result, even if incorrect or it takes a long time, is considered to have liveness.
+The property of a distributed system is that it will eventually come to some sort of consensus. A system stuck in an infinite loop would not be considered live, even if computations are taking place; a system that eventually provides a result, even if incorrect or it takes a long time, is considered to have liveness.
 
 ## Node explorer
```
Comment (xcthulhu) [inline docs/glossary.md:67]:
```suggestion
## Node Explorer
```

File: docs/glossary.md [hunk 6]
```diff
@@ -23,11 +23,11 @@ Byzantine fault tolerance (BFT) is a characteristic of a distributed system that
 
 ## Consensus
 
-Consensus is the process of a group of entities to agree on a particular data value (such as the ordering and makeup of blocks on a blockchain).  There are a variety of algorithms used for determining consensus.  GGX uses hybrid consensus - BABE for block production and GRANDPA as a finality gadget.
+Consensus is the process of a group of entities agreeing on a particular data value (such as the ordering and makeup of blocks on a blockchain). There are a variety of algorithms used for determining consensus. GGX uses hybrid consensus - BABE for block production and GRANDPA as a finality gadget.
 
 ## Controller account
```
Comment (xcthulhu) [inline docs/glossary.md:28]:
```suggestion
## Controller Account
```

File: docs/glossary.md [hunk 7]
```diff
@@ -23,11 +23,11 @@ Byzantine fault tolerance (BFT) is a characteristic of a distributed system that
...
 
-In Golden gate, a controller account is a type of account that is used to control the staking and nomination activities of a stash account. A stash account is the primary account that holds a user's GGX tokens and is used for staking and nominating, while the controller account is a separate account that is linked to the stash account to manage its staking and nomination activities.
+In Golden Gate, a controller account is a type of account that is used to control the staking and nomination activities of a stash account. A stash account is a primary account that holds a user's GGX tokens and is used for staking and nominating, while the controller account is a separate account that is linked to the stash account to manage its staking and nomination activities.
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply docs/glossary.md:30]: Also, there should be a hyperlink to the stash account.

File: SUMMARY.md [hunk 8]
```diff
@@ -1,6 +1,11 @@
 # Summary
  
-* [Consensus](docs/consensus.md)
+* Architecture
+    - Roles and responsibilities
+    - Accounts
+    - Generic transactions
```
Comment (xcthulhu) [inline SUMMARY.md:6]:
```suggestion
    - Generic Transactions
```

File: SUMMARY.md [hunk 9]
```diff
@@ -1,6 +1,11 @@
...
+    - Cross-chain transactions
```
Comment (xcthulhu) [inline SUMMARY.md:7]:
```suggestion
    - Cross-chain Transactions
```

File: SUMMARY.md [hunk 10]
```diff
@@ -1,6 +1,11 @@
 # Summary
  
-* [Consensus](docs/consensus.md)
+* Architecture
+    - Roles and responsibilities
```
[the remark this answers is not in the document]
Comment (FieldmanJR1) [reply SUMMARY.md:4]: Fixed capitalization in Summary.md

File: docs/glossary.md [hunk 11]
```diff
@@ -116,19 +112,19 @@ A session "key" is a BLS key for GRANDPA, a sr25519 key for BABE, and eventually
 
 ## Staking
 
-"Reserving" tokens (for GGX network, GGX tokens) which are put up as "collateral" for a chance to produce a valid block (and thus obtain a block reward).  Validators and nominators (who back validators through NPoS) together stake their GGX tokens in order to add blocks.
+"Reserving" tokens (for GGX network, GGX tokens) that are put up as "collateral" for a chance to produce a valid block (and thus obtain a block reward). Validators and nominators (who back validators through NPoS) together stake their GGX tokens in order to add blocks.
```
[the remark this answers is not in the document]
Comment (FieldmanJR1) [reply docs/glossary.md:115]: Fixed in branch fix-glossary-comments-issue6

File: docs/glossary.md (hunk 2 above)
Comment (FieldmanJR1) [reply docs/glossary.md:81]: Fixed in branch fix-glossary-comments-issue6

File: docs/glossary.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (FieldmanJR1) [reply docs/glossary.md:30]: Fixed in branch fix-glossary-comments-issue6

File: docs/glossary.md [hunk 12]
```diff
@@ -23,11 +23,11 @@ Byzantine fault tolerance (BFT) is a characteristic of a distributed system that
 
 ## Consensus
 
-Consensus is the process of a group of entities to agree on a particular data value (such as the ordering and makeup of blocks on a blockchain).  There are a variety of algorithms used for determining consensus.  GGX uses hybrid consensus - BABE for block production and GRANDPA as a finality gadget.
+Consensus is the process of a group of entities agreeing on a particular data value (such as the ordering and makeup of blocks on a blockchain). There are a variety of algorithms used for determining consensus. GGX uses hybrid consensus - BABE for block production and GRANDPA as a finality gadget.
```
[the remark this answers is not in the document]
Comment (FieldmanJR1) [reply docs/glossary.md:26]: Fixed in branch fix-glossary-comments-issue6

[the remark this answers is not in the document]
Comment (xcthulhu) [reply docs/glossary.md]: Merged branch

File: docs/glossary.md (hunk 1 above)
Comment (xcthulhu) [reply docs/glossary.md]: Merged branch

File: SUMMARY.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (xcthulhu) [reply SUMMARY.md]: I ended up just getting rid of these table of contents stubs.

File: docs/glossary.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (xcthulhu) [reply docs/glossary.md]: Merged into this branch

**Pull request merged**